wrappers/Lua/Makefile

```diff
@@ -0,0 +1,36 @@
+include lualib.mk
+
+COOLPROP_CFLAGS  ?= $(shell $(PKGCONFIG) --cflags CoolProp)
+COOLPROP_LDFLAGS ?= $(shell $(PKGCONFIG) --libs-only-L CoolProp)
+COOLPROP_LDLIBS  ?= $(or $(shell $(PKGCONFIG) --libs-only-l CoolProp), -lCoolProp)
+COOLPROP_INCDIR  ?= $(shell $(PKGCONFIG) --variable=includedir CoolProp)
+COOLPROP_LIBDIR  ?= $(shell $(PKGCONFIG) --variable=libdir CoolProp)
+COOLPROP_HEADER  ?= $(or $(COOLPROP_INCDIR)
```

Comment:
Yes, point taken. I edited my comment shortly after posting it. The line in question isn't actually used, so the fix would just be to remove it.